packages/design-system/src/components/Button/style.scss

```diff
@@ -131,9 +134,29 @@
 
   .btn__loader {
     position: absolute;
-    top: 50%;
-    transform: translate(-50%, -50%);
-    left: 50%;
+    top: 0;
+    left: 0;
+    opacity: 1;
+    width: 100%;
+    height: 100%;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    box-sizing: border-box;
+    padding: inherit;
+  }
+
+  .btn__loader-spinner {
+    min-width: 15px;
```

Comment:
Why do you need it? Small loader has `width: 15px` by default.

Reply:
<img width="172" alt="Design System - LiveChat API Developers Docs and Guides 2020-04-28 13-20-45" src="https://user-images.githubusercontent.com/24607370/80481614-17a1ec00-8953-11ea-8099-5d0eb95e260e.png">
Ok, I understand xD I used width 100px for button and its height changed also ;/

Reply:
Looks like you found a bug in Loader with label. Please fix the Loader component spinner by adding `flex-shrink: 0;` to `.lc-loader-spinner-wrapper` should solve issue for all loaders

Reply:
done